.github/workflows/ci.yml

```diff
@@ -183,6 +236,51 @@ jobs:
       - name: Stop sccache server
         run: sccache --stop-server
 
+  doc_test_mac:
+    name: Doc Test Mac
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [macos-latest]
+        rust: [stable]
+    env:
+      RUST_BACKTRACE: full
+      RUSTC_WRAPPER: sccache
+      SCCACHE_CACHE_SIZE: 300M
+      SCCACHE_DIR: /Users/runner/Library/Caches/Mozilla.sccache
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install sccache
+        run: brew install sccache
+      - name: Install ${{ matrix.rust }}
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: ${{ matrix.rust }}
+          override: true
+      - name: Cache cargo registry
+        uses: actions/cache@v2
+        continue-on-error: true
+        with:
+          path: ~/.cargo/registry/cache
+          key: test-${{ runner.os }}-${{ matrix.conf }}-cargo-registry-${{ hashFiles('**/Cargo.lock') }}
+          restore-keys: |
+            test-${{ runner.os }}-${{ matrix.conf }}-cargo-registry-
+      - name: Cache sccache output
+        uses: actions/cache@v2
+        continue-on-error: true
+        with:
+          path: /Users/runner/Library/Caches/Mozilla.sccache
+          key: test-${{ runner.os }}-${{ matrix.conf }}-sccache-${{ hashFiles('**/Cargo.*') }}
+          restore-keys: |
+            test-${{ runner.os }}-${{ matrix.conf }}-sccache-
+      - name: Start sccache server
+        run: sccache --start-server
+      - name: Run doc tests
+        run: make run-all-doc-test
+      - name: Stop sccache server
+        run: sccache --stop-server
+
```

Comment:
since Mac/Linux are similar.  Can we combine into a single job?

Reply:
I'll can combine them. The only reason they were separate was to keep consistency with the unit tests.